### ISSUE TITLE
Layout displays wrongly.

### ISSUE THREAD

**Issue by fifiteen82726** [UiReady/gsdk#1]
I did every step on README, is that the final result? Cuz I thought the background picture and slogan text encountered some trouble, did I miss something?
Picture: ![Alt Text](http://i.imgur.com/Zi1wIjY.png)

**Comment by timshingyu** [comment]
The background picture look correct. However, the slogan text should look like this

![screen shot 2015-11-25 at 8 47 31 pm](https://cloud.githubusercontent.com/assets/190269/11397650/f8ea7dc8-93b5-11e5-8e96-d3171f41e694.png)

Your slogan seem overlapped. Did you included any other css in your application?

**Comment by fifiteen82726** [comment]
Nope, I haven't put any css file into there, I just created a new app that was followed the steps. I turn off the font-size, and they didn't overlay. 
![Alt Text](http://i.imgur.com/YHAvnEg.png)

**Comment by timshingyu** [comment]
Can you try another browser to view again?
If you using google chrome, please try disable all google chrome extensions to view it again.
